iset.mm

```diff
@@ -99758,6 +99758,64 @@ A Cauchy sequence (as defined here, which has a rate of convergence
     LXCSVPWEXESVGURVHVPVRWAVPVQCDZVRNDABVIZVQPQVPVSVTWTXAUMVPXJTVRIJXKVQVJQVPVS
     VTWTXAVPVNTVSIJWOAVJQVPVOTVTIJWPBVJQVKVLVM $.
 
+  $( An upper bound for ` { A , B } ` .
+     (Contributed by Jim Kingdon, 20-Dec-2021.) $)
+  maxabsle $p |- ( ( A e. RR /\ B e. RR ) ->
+      A <_ ( ( ( A + B ) + ( abs ` ( A - B ) ) ) / 2 ) ) $=
+    ? $.
+
+  $( The maximum of two reals is no smaller than the first real.
+     Lemma 3.10 of [Geuvers], p. 10.
+     (Contributed by Jim Kingdon, 21-Dec-2021.) $)
+  maxle1 $p |- ( ( A e. RR /\ B e. RR ) -> A <_ sup ( { A , B } , RR , < ) ) $=
+    ? $.
+
+  $( The maximum of two reals is no smaller than the second real.
+     Lemma 3.10 of [Geuvers], p. 10.
+     (Contributed by Jim Kingdon, 21-Dec-2021.) $)
+  maxle2 $p |- ( ( A e. RR /\ B e. RR ) -> B <_ sup ( { A , B } , RR , < ) ) $=
+    ? $.
```

Comment:
maxle2 is a consequence of maxle1 and maxcom (probably shorter proof), so maybe put maxcom at the very top.